docs/can-guides/commitment/recipes/canvas-clock/canvas-clock.md

```diff
@@ -27,22 +158,61 @@ The following sections are broken down into the following parts:
 
 ## Setup ##
 
-__START THIS TUTORIAL BY CLONING THE FOLLOWING JS BIN__:
-
-> Click the **JS Bin** button.  The JS Bin will open in a new window. In that new window, under `File`, click `Clone`.
-
-<a class="jsbin-embed" href="https://jsbin.com/jeligek/1/embed?html,css,js,output">
-  CanJS Canvas Clock on jsbin.com
-</a>
+__START THIS TUTORIAL BY CLONING THE FOLLOWING CODEPEN__:
```

Comment:
This should match the “START THIS TUTORIAL BY CLICKING” text shown in this guide: http://localhost:8080/doc/guides/recipes/signup-simple.html